src/components/button/Button.jsx

```diff
@@ -4,7 +4,8 @@ import classNames from 'classnames';
 
 export const Button = ({
   children,
-  secondary,
+  variant,
+  type,
```

Comment:
You could default `type` here to `type = "button"` then you don't have to do the ternary `type={type ? type : "button"}`, it can just be `type={type}`